test/performance/shmem_perf_suite/latency_common.h

```diff
@@ -31,57 +31,31 @@
 #define GET_IO_NODE !PUT_IO_NODE
 #define INIT_VALUE 1
 
-#define MAX_MSG_SIZE (1<<23)
-#define START_LEN 1
-
-#define INC 2
-#define TRIALS 100
-#define WARMUP 10
-
-typedef struct perf_metrics {
-   unsigned int start_len, max_len;
-   unsigned int inc, trials;
-   unsigned int warmup;
-   int validate;
-   int my_node, npes;
-   long * target;
-   char * src, *dest;
-} perf_metrics_t;
-
-static void data_init(perf_metrics_t * data) {
-   data->start_len = START_LEN;
-   data->max_len = MAX_MSG_SIZE;
-   data->inc = INC;
-   data->trials = TRIALS;
-   data->warmup = WARMUP; /*number of initial iterations to skip*/
-   data->validate = false;
-   data->my_node = shmem_my_pe();
-   data->npes = shmem_n_pes();
-   data->target = NULL;
-   data->src = NULL;
-   data->dest = NULL;
+static 
+void init_metrics(perf_metrics_t *metric_info) {
+    metric_info->t_type = LAT;
+    set_metric_defaults(metric_info);
+    metric_info->target = NULL;
 }
 
-static inline void print_results_header(void) {
-   printf("\nLength                  Latency                       \n");
-   printf("in bytes            in micro seconds              \n");
+static inline 
+void print_latency_header(perf_metrics_t metric_info) {
```

Comment:
no need to include metric_info for this call

Reply:
Thanks @kseager. Will remove it for now. 
I kept it initially to print with min, max, and avg options, if we plan to add it. But, this PR is out of scope for that.